zerver/views/home.py

```diff
@@ -75,9 +75,8 @@ def approximate_unread_count(user_profile):
         flags=UserMessage.flags.read).count()
 
 def sent_time_in_epoch_seconds(user_message):
-    # type: (UserMessage) -> Optional[float]
-    # user_message is a UserMessage object.
```

Comment:
:-D I appreciate when ad-hoc type annotations like this can just go away.

(In this case it already could have with the existing annotation; presumably we just missed it then.)